packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -26,18 +26,12 @@ import { GridColumns } from 'packages/grid/_modules_/grid/models/colDef/gridColD
 
 const SPACE_KEY = { key: ' ' };
 const SHIFT_SPACE_KEY = { ...SPACE_KEY, shiftKey: true };
+const NO_LAYOUT = /jsdom/.test(window.navigator.userAgent);
```

Comment:
Why not name it `isJsDom`?

Reply:
NO_LATOUT explains what the variable is meant to do. There are three different notions:

- is layouting available?
- jsdom doesn't have layouting
- when layouting is not available, we need to disable virtualization

Reply:
ok, `NO_LAYOUT` is not ideal as a var name. I would rename to `HAS_LAYOUT`

```tsx
const HAS_LAYOUT = !/jsdom/.test(window.navigator.userAgent); 
```

Reply:
This would yield to a double negation everywhere, not great. Please provide an alternative that is true if it doesn't have the layout available.